handlers/user_handler.go

```diff
@@ -25,20 +29,20 @@ func (h *Handler) UserHandler(w http.ResponseWriter, req *http.Request, _ *model
 }
 
 // swagger:route GET /api/user/prefs UserAPI idGetAnonymousStats
-// Handle GET for anonymous stats
+// Handle GET for anonymous stats or Load Test Preferences
 //
-// Returns anonymous stats for user
+// Returns anonymous stats for user and Load Test Preferences
 // responses:
 // 	200: anonymousStatsResponseWrapper
 
 // swagger:route POST /api/user/prefs UserAPI idPostAnonymousStats
-// Handle GET for anonymous stats
+// Handle GET for anonymous stats or Load Test Preferences
 //
-// Updates anonymous stats for user
+// Updates anonymous stats for user or Update load test preferences
 // responses:
 // 	200: anonymousStatsResponseWrapper
 
-// AnonymousStatsHandler updates anonymous stats for user
+// AnonymousStatsHandler updates anonymous stats for user or for persisting load test preferences
 func (h *Handler) AnonymousStatsHandler(w http.ResponseWriter, req *http.Request, prefObj *models.Preference, user *models.User, provider models.Provider) {
```

Comment:
Change handler name to a more generic one? Something like `userPrefHandler` ? Just a suggestion.

Reply:
@kumarabd @navendu-pottekkat @leecalcote views??